doc/book/sql-ddl.md

```diff
@@ -87,8 +87,21 @@ $table->changeColumn('name', Column\Varchar('new_name', 50));
 You may also *drop* existing columns or constraints:
 
 ```php
+use \Zend\Db\Metadata\Object\ConstraintObject;
+
 $table->dropColumn('foo');
-$table->dropConstraint('my_index');
+
+$constraint = new ConstraintObject('my_index', null);
+$table->dropConstraint($constraint);
+```
+
+Notice: On MySQL, you need to specify the type of constraint you want to drop:
+```php
+use \Zend\Db\Metadata\Object\ConstraintObject;
+
+$fkConstraint = new ConstraintObject('my_fk', null);
+$fkConstraint->setType('FOREIGN KEY');
```

Comment:
I see... In that case metadata analyzer is broken not assigning foreign key constraint to ForeignKey constraint object. I should not be penalized for wanting to use given by the framework ForeignKey object to describe foreign key constraint. I suppose that means both need to be handled by the Decorator at this point since that's what the library gives us. Or clean creation of ConstraintObject to create ForeignKey,  which would be further reaching change.